docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.WifiState.EventName`            | `SystemEvents.WifiState.StateKey`             | - `SystemEvents.WifiState.StateValueOn`:  Wi-Fi on <br>-  `SystemEvents.WifiState.StateValueOff`: Wi-Fi off <br>- `SystemEvents.WifiState.StateValueConnected`: Wi-Fi connection established | When the Wi-Fi  state has changed.       | You can get the  current value with the `Network.Connection.ConnectionManager.WiFiState` property. |
+    | `SystemEvents.Btstate.EventName`              | `SystemEvents.Btstate.StateKey`               | - `SystemEvents.Btstate.StateValueOff`:  Legacy Bluetooth off <br>-  `SystemEvents.Btstate.StateValueOn`: Legacy Bluetooth on | When the  Bluetooth state has changed.   | You can get the  current value with the `Network.Bluetooth.BluetoothAdapter.IsBluetoothEnabled` property. |
+    | `SystemEvents.Btstate.EventName`              | `SystemEvents.Btstate.LeStateKey`             | - `SystemEvents.Btstate.LeStateValueOff`:  LE function off <br>- `SystemEvents.Btstate.LeStateValueOn`: LE function on | When Bluetooth  LE state has changed.    |   -                                       |
+    | `SystemEvents.Btstate.EventName`              | `SystemEvents.Btstate.TransferStateKey`       | - `SystemEvents.Btstate.TransferStateValueNontransfering`:  Idle state  <br>-  `SystemEvents.Btstate.TransferStateValueTransfering`: File is transferring | When the file  transfer state has changed. | -                                         |    
+
+- libslp-location
+
+  - location
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.LocatingEnableState.EventName` | `SystemEvents.LocatingEnableState.StateKey`   | - `SystemEvents.LocatingEnableState.StateValueDisabled`:  Location disabled <br>-  `SystemEvents.LocatingEnableState.StateValueEnabled`: Location enabled | When the `location-enable_state` has been  changed, for example, by the user toggling the location setting in the  settings menu or quick panel. | You can get the  current value with the `Location.LocatorHelper.IsEnabledType` property. |
+    | `SystemEvents.GpsEnableState.EventName`      | `SystemEvents.GpsEnableState.StateKey`        | - `SystemEvents.GpsEnableState.StateValueDisabled`:  GPS disabled  <br>- `SystemEvents.GpsEnableState.StateValueEnabled`: GPS enabled | When the `gps-enable_state` has changed.   | You can get the  current value with the `Location.LocatorHelper.IsEnabledType` property. |
+    | `SystemEvents.NpsEnableState.EventName`      | `SystemEvents.NpsEnableState.StateKey`        | - `SystemEvents.NpsEnableState.StateValueDisabled`:  NPS disabled <br>-  `SystemEvents.NpsEnableState.StateValueEnabled`: NPS enabled | When the NPS  setting has been changed, for example, by the user toggling the location  settings. | You can get the  current value with the `Location.LocatorHelper.IsEnabledType` property. |
```

Comment:
@divyk In this PR, in the tables, "has been changed" and "has changed" are mixed. Please check which one is correct to be used and guide to @kilig.